backend/src/controllers/rentalAgency.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+const RentalAgencyModel = require('../models/rentalAgency');
+
+
+const create = (req, res) => {
+    if (Object.keys(req.body).length === 0) return res.status(400).json({
+        error: 'Bad Request',
+        message: 'The request body is empty'
+    });
+
+    RentalAgencyModel.create(req.body)
+        .then(tour => res.status(201).json(tour))
+        .catch(error => res.status(500).json({
+            error: 'Internal server error',
+            message: error.message
+        }));
+};
+
+const list  = (req, res) => {
```

Comment:
List is cool, but not that useful, we need node, which will return rentalAgencies around some location.

Reply:
Changed to return nearby rental agencies